src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptor.java

```diff
@@ -110,7 +111,8 @@
                 return super.onMethodCall(invoker, receiver, method, args);
             }
 
-            throw new RejectedAccessException("unclassified method " + EnumeratingWhitelist.getName(receiver.getClass()) + " " + method + printArgumentTypes(args));
+            // no such method exists
+            throw new MissingMethodException(method, receiver.getClass(), args);
```

Comment:
What about `MissingFieldException` and `MissingPropertyException`? (I do not think `MissingClassException` is relevant here.)